services/api/db/admin.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib import admin
+from db.models import User
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+
+class UserModelAdmin(BaseUserAdmin):
+  # The fields to be used in displaying the User model.
+  # These override the definitions on the base UserModelAdmin
+  # that reference specific fields on auth.User.
+  list_display = ('id', 'email', 'name', 'is_verified', 'is_admin', "auth_provider")
+  list_filter = ('is_admin',)
+  fieldsets = (
+      ('User Credentials', {'fields': ('email', 'password')}),
+      ('Personal info', {'fields': ('name',)}),
+      ('Permissions', {'fields': ('is_admin',)}),
+  )
+  # add_fieldsets is not a standard ModelAdmin attribute. UserModelAdmin
+  # overrides get_fieldsets to use this attribute when creating a user.
+  add_fieldsets = (
+      (None, {
+          'classes': ('wide',),
+          'fields': ('email', 'name', 'password', 'confirm_password'),
+      }),
+  )
+  search_fields = ('email',)
+  ordering = ('email', 'id')
+  filter_horizontal = ()
```

Comment:
```suggestion
  filter_horizontal = ()
```
should the fields be specified in the filter_horizontal() ?

Reply:
resolved that issue